### PR TITLE
Add charts for weight, height and bmi

### LINKED ISSUE #98: Add charts for statistics (height, weight)

### PR DESCRIPTION
**Pull request by divakarmal**
Closes #98

### CODE REVIEW

File: src/main/resources/view/PatientProfile.fxml [hunk 1]
```diff
@@ -1,20 +1,25 @@
...
+              <items>
+                <AnchorPane fx:id="medicalNotesPanel" minHeight="0.0" minWidth="0.0" prefHeight="793.0" prefWidth="396.0" />
+                <AnchorPane fx:id="patientChartCard" minHeight="0.0" minWidth="0.0" prefHeight="398.0" prefWidth="317.0" />
+              </items>
+            </SplitPane>
             </AnchorPane>
-          <AnchorPane fx:id="medicalNotesPanel" minHeight="0.0" minWidth="0.0" prefHeight="160.0" prefWidth="100.0" />
         </items>
     </SplitPane>
 </AnchorPane>
```
Comment (divakarmal) [inline src/main/resources/view/PatientProfile.fxml]: Changes here will need to be edited later when we adjust layout of patient profile page

File: src/main/java/seedu/medibook/ui/patientprofile/PatientChartCard.java [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+    /**
+     * Creates a {@code PatientChartCard} for the given {@code Patient}.
+     */
+    public PatientChartCard(Patient patient) {
+        super(FXML);
+        this.patient = patient;
+        cardHeader.setText("Chart: ");
+        weightChart.setTitle("Weight Chart");
+        heightChart.setTitle("Height Chart");
+        bmiChart.setTitle("BMI Chart");
```
Comment (foojingyi) [inline src/main/java/seedu/medibook/ui/patientprofile/PatientChartCard.java:53]: Is it possible to abstract out some parts of this constructor?

**Review by foojingyi — APPROVED** [review]
Constructor looks a bit long. Other than that lgtm~!

File: src/main/java/seedu/medibook/ui/patientprofile/PatientChartCard.java [hunk 3]
```diff
@@ -0,0 +1,200 @@
...
+import javafx.scene.chart.NumberAxis;
+import javafx.scene.chart.XYChart;
+import javafx.scene.control.Label;
+import javafx.scene.layout.Region;
+import seedu.medibook.model.commonfields.Date;
+import seedu.medibook.model.patient.Height;
+import seedu.medibook.model.patient.Patient;
+import seedu.medibook.model.patient.Weight;
+import seedu.medibook.ui.UiPart;
+
```
Comment (foojingyi) [inline src/main/java/seedu/medibook/ui/patientprofile/PatientChartCard.java:24]: May want to add JavaDoc comments to this class?

**Pull request merged**